viewing_party/party.py

```diff
@@ -0,0 +1,264 @@
+import statistics
+
+# Wave 1
+def create_movie(title, genre, rating):
+    """
+    Takes in three parameters: title, genre,
+    and rating to build and return dictionary movie.
+    If any of these parameters are falsy, returns None.
+    """
+    if title and genre and rating:
+        movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating,
+             }
+        return movie
```

Comment:
this is nice and clean! one thing to note is that when a function has no explicit return statement, a function returns a default value None. So while this works, in the future think about adding an explicit `return None` so that other coders know you want that to happen!

```suggestion
    if title and genre and rating:
        movie = {
            "title": title,
            "genre": genre,
            "rating": rating,
             }
        return movie
    else:
        return None
```